--- forkSrcPrefix/kernel/conf/account.go	2026-09-07 22:14:30
+++ forkDstPrefix/kernel/conf/account.go	2026-09-07 22:14:34
@@ -23,7 +23,7 @@
 
 func NewAccount() *Account {
 	return &Account{
-		DisplayTitle: true,
-		DisplayVIP:   true,
+		DisplayTitle: false,
+		DisplayVIP:   false,
 	}
 }
--- forkSrcPrefix/kernel/conf/appearance.go	2026-09-07 22:14:30
+++ forkDstPrefix/kernel/conf/appearance.go	2026-09-07 22:14:34
@@ -52,8 +52,8 @@
 		Icon:                "litheness",
 		CodeBlockThemeLight: "github",
 		CodeBlockThemeDark:  "base16/dracula",
-		Lang:                "en",
-		CloseButtonBehavior: 0,
+		Lang:                "zh_CN",
+		CloseButtonBehavior: 1,
 		HideToolbar:         true,
 		HideStatusBar:       false,
 		StatusBar:           util.NewStatusBar(util.IsMobileContainer()),
--- forkSrcPrefix/kernel/conf/sync.go	2026-09-07 22:14:30
+++ forkDstPrefix/kernel/conf/sync.go	2026-09-07 22:14:34
@@ -39,8 +39,8 @@
 		Enabled:             false,
 		Perception:          false,
 		Mode:                1,
-		GenerateConflictDoc: false,
-		Provider:            ProviderSiYuan,
+		GenerateConflictDoc: true,
+		Provider:            ProviderS3,
 		Interval:            30,
 		LAN:                 &LANSync{MaxConcurrentReqs: 16},
 	}
