diff --git forkSrcPrefix/kernel/conf/account.go forkDstPrefix/kernel/conf/account.go
index c4590e2..133208f 100644
--- forkSrcPrefix/kernel/conf/account.go
+++ forkDstPrefix/kernel/conf/account.go
@@ -23,7 +23,7 @@
 
 func NewAccount() *Account {
 	return &Account{
-		DisplayTitle: true,
-		DisplayVIP:   true,
+		DisplayTitle: false,
+		DisplayVIP:   false,
 	}
 }
diff --git forkSrcPrefix/kernel/conf/appearance.go forkDstPrefix/kernel/conf/appearance.go
index 1bc316a..fd9f81a 100644
--- forkSrcPrefix/kernel/conf/appearance.go
+++ forkDstPrefix/kernel/conf/appearance.go
@@ -50,8 +50,8 @@
 		Icon:                "litheness",
 		CodeBlockThemeLight: "github",
 		CodeBlockThemeDark:  "base16/dracula",
-		Lang:                "en",
-		CloseButtonBehavior: 0,
+		Lang:                "zh_CN",
+		CloseButtonBehavior: 1,
 		HideToolbar:         true,
 		HideStatusBar:       false,
 		StatusBar:           util.NewStatusBar(util.IsMobileContainer()),
diff --git forkSrcPrefix/kernel/conf/sync.go forkDstPrefix/kernel/conf/sync.go
index b076c5e..662b300 100644
--- forkSrcPrefix/kernel/conf/sync.go
+++ forkDstPrefix/kernel/conf/sync.go
@@ -38,8 +38,8 @@
 		Enabled:             false,
 		Perception:          false,
 		Mode:                1,
-		GenerateConflictDoc: false,
-		Provider:            ProviderSiYuan,
+		GenerateConflictDoc: true,
+		Provider:            ProviderS3,
 		Interval:            30,
 		LAN:                 &LANSync{MaxConcurrentReqs: 16},
 	}
