diff --git forkSrcPrefix/kernel/api/system.go forkDstPrefix/kernel/api/system.go
index bb3aa78..837b791 100644
--- forkSrcPrefix/kernel/api/system.go
+++ forkDstPrefix/kernel/api/system.go
@@ -274,16 +274,7 @@ func addCustomEmoji(name string, items *[]map[string]any) {
 }
 
 func checkUpdate(c *gin.Context) {
-	ret := gulu.Ret.NewResult()
-	defer c.JSON(http.StatusOK, ret)
-
-	arg, ok := util.JsonArg(c, ret)
-	if !ok {
-		return
-	}
-
-	showMsg := arg["showMsg"].(bool)
-	model.CheckUpdate(showMsg)
+	_ = c
 }
 
 func exportLog(c *gin.Context) {
diff --git forkSrcPrefix/kernel/conf/system.go forkDstPrefix/kernel/conf/system.go
index 60ccf4f..d8682bf 100644
--- forkSrcPrefix/kernel/conf/system.go
+++ forkDstPrefix/kernel/conf/system.go
@@ -57,7 +57,7 @@ func NewSystem() *System {
 		Name:               util.GetDeviceName(),
 		KernelVersion:      util.Ver,
 		NetworkProxy:       &NetworkProxy{},
-		DownloadInstallPkg: true,
+		DownloadInstallPkg: false,
 	}
 }
 
diff --git forkSrcPrefix/kernel/model/mount.go forkDstPrefix/kernel/model/mount.go
index b34f88c..e3f8a9e 100644
--- forkSrcPrefix/kernel/model/mount.go
+++ forkDstPrefix/kernel/model/mount.go
@@ -276,8 +276,8 @@ func Mount(boxID string) (alreadyMount bool, err error) {
 		task.AppendAsyncTaskWithDelay(task.PushMsg, 3*time.Second, util.PushErrMsg, Conf.Language(244), 7000)
 		go func() {
 			// 每次打开帮助文档时自动检查版本更新并提醒 https://github.com/siyuan-note/siyuan/issues/5057
-			time.Sleep(time.Second * 10)
-			CheckUpdate(true)
+			// time.Sleep(time.Second * 10)
+			// CheckUpdate(true)
 		}()
 	}
 
diff --git forkSrcPrefix/kernel/model/updater.go forkDstPrefix/kernel/model/updater.go
index d1773fb..cbc13a8 100644
--- forkSrcPrefix/kernel/model/updater.go
+++ forkDstPrefix/kernel/model/updater.go
@@ -312,6 +312,10 @@ func isVersionUpToDate(releaseVer string) bool {
 var skipInstallPkgPlatformCached = -1
 
 func skipNewVerInstallPkg() bool {
+	// Conf.System.DownloadInstallPkg(自动下载更新安装包) 强制赋值为false
+	Conf.System.DownloadInstallPkg = false
+	Conf.Save()
+
 	if skipInstallPkgPlatformCached == -1 {
 		skipInstallPkgPlatformCached = 0
 		if !gulu.OS.IsWindows() && !gulu.OS.IsDarwin() {
