diff --git forkSrcPrefix/kernel/conf/system.go forkDstPrefix/kernel/conf/system.go
index 8bf0031..8fd9ba5 100644
--- forkSrcPrefix/kernel/conf/system.go
+++ forkDstPrefix/kernel/conf/system.go
@@ -77,7 +77,7 @@ func NewSystem() *System {
 		Name:               util.GetDeviceName(),
 		KernelVersion:      util.Ver,
 		NetworkProxy:       &NetworkProxy{},
-		DownloadInstallPkg: true,
+		DownloadInstallPkg: false,
 		UpdateChannel:      UpdateChannelStable,
 	}
 }